pkg/containerwatcher/v1/container_watcher.go

```diff
@@ -105,6 +114,30 @@ func CreateIGContainerWatcher(cfg config.Config, applicationProfileManager appli
 		return nil, fmt.Errorf("creating open worker pool: %w", err)
 	}
 
+	// Create a network worker pool
+	networkWorkerPool, err := ants.NewPoolWithFunc(networkWorkerPoolSize, func(i interface{}) {
+		event := i.(tracernetworktype.Event)
+
+		networkEvent := &networkmanager.NetworkEvent{
+			Port:     event.Port,
+			Protocol: event.Proto,
+			PktType:  event.PktType,
+			Destination: networkmanager.Destination{
+				Namespace: event.DstEndpoint.Namespace,
+				Name:      event.DstEndpoint.Name,
+				Kind:      networkmanager.EndpointKind(event.DstEndpoint.Kind),
+				IPAddress: event.DstEndpoint.Addr,
+			},
+		}
+		networkEvent.SetPodLabels(event.PodLabels)
+		networkEvent.SetDestinationPodLabels(event.DstEndpoint.PodLabels)
+
+		networkManagerClient.SaveNetworkEvent(event.K8s.ContainerName, event.K8s.PodName, networkEvent)
```

Comment:
If we disable we won't be getting network events , no?

Reply:
Today yes because only your manager is interested. However if we add another one we could have this situation... Maybe just put a FIXME comment for the moment to not forget it